.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,31 +16,20 @@ We love pull requests that resolve an open Crossplane issue. If yours does, you
 can uncomment the below line to indicate which issue your PR fixes, for example
 "Fixes #500":
 
-Fixes #
 -->
+Fixes #
 
-### How has this code been tested?
-<!--
-Before reviewers can be confident in the correctness of a pull request,
-it needs to tested and shown to be correct. In this section, briefly
-describe the testing that has already been done or which is planned.
--->
+I have:
+
+- [ ] Read and followed Crossplane's [contribution process].
```

Comment:
what do you think about also linking directly to the summary checklist for this guide? so folks are encouraged to read the whole guide with the current link you have, but also have easy access to the quick check list too.

we could also still include the summary checklist inline in this template so it's more visible. what do you think?

Reply:
I'd prefer to leave this as is.

My feeling is that the summary 'checklist' is too long to expect folks to really pay attention to and check off with every PR. I suspect it would quickly become ignored, or blindly ticked off. I'd like to try keeping it _really_ short - basically validate that the PR is likely to build, and have the author acknowledge that they're familiar with the project's expectations.